src/bun.js/bindings/BunProcess.h

```diff
@@ -36,6 +36,7 @@ class Process : public WebCore::JSEventEmitter {
     }
 
     DECLARE_EXPORT_INFO;
+    bool m_reportOnUncaughtException;
```

Comment:
Let's default initialize this to false to avoid undefined memory
```suggestion
    bool m_reportOnUncaughtException = false;
```